tensorflow/lite/micro/micro_allocator.cc

```diff
@@ -68,7 +69,7 @@ class MicroBuiltinDataAllocator : public BuiltinDataAllocator {
   TF_LITE_REMOVE_VIRTUAL_DELETE
 
  private:
-  SimpleMemoryAllocator* memory_allocator_;
+  IPersistentBufferAllocator* memory_allocator_;
```

Comment:
Similar to the previous question - does it make sense to name it "persistent_memory_allocator_"?

Reply:
Agreed. Changed